requirements_all_ds.txt

```
@@ -42,3 +42,4 @@ xlrd==2.0.1
 openpyxl==3.0.7
 firebolt-sdk
 pandas==1.3.4
+nzpy
```

Mirrors what was added to `requirements.txt` originally.

```suggestion
nzpy>=1.15
nzalchemy
```